src/VisualStudio/Core/Next/Remote/JsonRpcSession.cs

```
@@ -52,9 +49,9 @@ internal class JsonRpcSession : RemoteHostClient.Session
 
             try
             {
-                await session.InitializeAsync().ConfigureAwait(false);
+                await session.InitializeAsync(cancellationToken).ConfigureAwait(false);
             }
-            catch when (!cancellationToken.IsCancellationRequested)
+            catch
```

Do we want to at least report the exception we got here?